src/service.rs

```diff
@@ -65,9 +72,10 @@ pub struct NodeConfig {
     pub mine: bool,
 }
 
-impl Default for NodeConfig {
+impl<F: substrate_service::ServiceFactory> Default for NodeConfig<F> {
```

Comment:
substrate_service::ServiceFactory is imported, no need add "substrate_service::"